Sources/DashX/DashXAppDelegate.swift

```diff
@@ -61,7 +65,11 @@ open class DashXAppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificat
 
         // Pass notification reciept information to Firebase
         Messaging.messaging().appDidReceiveMessage(message)
-
+        
+        if let id = dashXClient.getNotificationID(message) {
+            dashXClient.trackNotification(id, .clicked, ISO8601DateFormatter.timeStamp)
```

Comment:
Can we create a method in DashXClient and use that instead?

```
trackNotification(message: message, event: .delivered/.clicked)
```

Reply:
created method for the same in dashxClient